content/intro-to-storybook/vue/es/deploy.md

```diff
@@ -0,0 +1,101 @@
+---
+title: 'Desplegar Storybook'
+tocTitle: 'Desplegar'
+description: 'Desplegar Storybook online con GitHub y Netlify'
+---
+
+En este tutorial hemos ejecutado Storybook en nuestra máquina de desarrollo. También se puede compartir ese Storybook con el equipo, especialmente con los miembros no técnicos. Afortunadamente, es fácil implementar Storybook en línea.
+
+<div class="aside">
+<strong>¿Hiciste los test con Chromatic antes?</strong>
+<br/>
+🎉 Sus historias ya están desplegadas! Chromatic indexa de forma segura sus historias en línea y las rastrea a través de ramas y commits. Salta este capítulo y ve a la <a href="/react/es/conclusion">conclusión</a>.
+</div>
+
+## Exportando como una app estática
+
+Para desplegar Storybook primero necesitamos exportarlo como una aplicación web estática. Esta funcionalidad ya está incorporada en Storybook, sólo necesitamos activarla añadiendo un script al `package.json`.
+
+```javascript
+{
+  "scripts": {
+    "build-storybook": "build-storybook -c .storybook -s public -o storybook-static"
+  }
+}
+```
+
+Ahora, cuando ejecutes Storybook a través de `yarn storybook`, obtendrás un Storybook estático en el directorio `storybook-static`.
+
+## Despliegue continuo
+
+Queremos compartir la última versión de los componentes cada vez que hagamos push del código. Para ello necesitamos desplegar de forma continua Storybook. Confiaremos en GitHub y Netlify para desplegar nuestro sitio estático. Estaremos usando el plan gratuito de Netlify.
+
+### GitHub
```

Comment:
This section needs a bit of work, once again the cli already setups a local repo so this is a bit outdated, as you're working already in the translation, see [this](https://deploy-preview-228--tender-kilby-05eed0.netlify.com/intro-to-storybook/angular/en/deploy/) and adjust accordingly to vue. The original doc will follow suit shortly.